From c0adaafa7275a397bf513d5ced20209d1633b3df Mon Sep 17 00:00:00 2001
From: Moritz Fischer <moritz.fischer@ettus.com>
Date: Fri, 31 Mar 2017 15:31:14 -0700
Subject: [PATCH 19/24] ARM: dt: zynq-ni-sulfur-rev3: Use eth_addrs in NVMEM
 provider

The MAC addresses are stored in an EEPROM that presents itself as NVMEM
provider, make use of it.

Signed-off-by: Moritz Fischer <moritz.fischer@ettus.com>
---
 arch/arm/boot/dts/zynq-ni-sulfur-rev3.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/zynq-ni-sulfur-rev3.dts b/arch/arm/boot/dts/zynq-ni-sulfur-rev3.dts
index b0469ad..eee3aec 100644
--- a/arch/arm/boot/dts/zynq-ni-sulfur-rev3.dts
+++ b/arch/arm/boot/dts/zynq-ni-sulfur-rev3.dts
@@ -284,6 +284,9 @@
 	phy-mode = "rgmii-id";
 	phy-handle = <&ethernet_phy0>;
 
+	nvmem-cells = <&eth0_addr>;
+	nvmem-cell-names = "address";
+
 	ethernet_phy0: ethernet-phy@0 {
 		reg = <0>;
 		reset-gpios = <&gpio0 50 GPIO_ACTIVE_LOW>;
-- 
2.7.4

